src/simplicial-hott/08-covariant.rzk.md

```diff
@@ -32,9 +32,9 @@ live over a specified arrow in the base type.
   (u : C x)          -- A lift of the domain.
   (v : C y)          -- A lift of the codomain.
   : U
-    := {t : 2 | Δ¹ t } -> C (f t)
-      [t === 0_2 |-> u ,
-      t === 1_2 |-> v ]
+    := {t : Δ¹ } -> C (f t)
```

Comment:
Elsewhere we've written ```(t : Δ¹)```. I don't have a strong preference for one or the other. Maybe in fact the {} braces make it clearer that such things are shapes not types?

Reply:
In fact why don't we go ahead and change all (variable : tope) to {variable : tope} for now to maintain consistency with the syntax being used for variable pairs?

Reply:
I suggest to go the other direction and remove the use of curly braces for parameters.
This way parameters are always in parentheses (including `(t : 2 | Δ¹ t)`), making code more uniform.
The only place where curly braces remain would be indices used for identity types: `=_{...}` and `refl_{...}`.
I have implemented support for parentheses (also supporting patterns in parameters) in the rzk v0.5.3 (latest release), with current syntax also supported.
If there is no objection to move away from curly braces, I will make a corresponding PR.

Reply:
> Maybe in fact the {} braces make it clearer that such things are shapes not types?

This was the original motivation for using curly braces, but there are a few issues:

1. Parentheses work just fine now, so using different punctuation is slightly confusing.
2. I think it is better to rely on semantic highlighting to help disambiguate cube/tope/type layers of the theory, as it is not always immediately clear from syntax and type checking is required. (that said, currently semantic highlighting is based on syntax only, but this is subject to improvement)
3. Shapes are also useful as types and there is a plan to introduce shape types, so curly braces lose significance, in a sense. Again, disambiguating shapes in the tope layer from shape types would only be possible with semantic highlighting (since I assume in syntax we would want to use the same syntax for both).
4. Curly braces are useful for other syntactic features (e.g. code blocks), so I would like to remove unnecessary meaning from `{` and `}`.
5. The tool that I use for parsing at the moment (BNF Converter) treats curly braces in a special way, which also motivates me to use them less for ad hoc purposes.

I would also say that I originally used `<` and `>` for extension types, to mimic syntax in the paper, but in practice it seems that those are completely redundant (and have been released to be used as proper comparison operators in the future).

Reply:
Switching to () works for me and is simpler overall. So long as it now works for all shapes I'm quite happy to move in that direction.